src/Microsoft.AspNetCore.Server.Kestrel/Http/Frame.cs

```diff
@@ -246,6 +248,8 @@ public void Reset()
 
             _manuallySetRequestAbortToken = null;
             _abortedCts = null;
+
+            _requestRejectReason = RequestRejectionReason.None;
```

Comment:
At some point when I was making theses changes this was necessary, but now it's just a leftover from a previous state of the code.